src/dune_rules/coq/coq_config.mli

```diff
@@ -1,24 +1,5 @@
 open Import
 
-(** Data of a Coq configuration. *)
-type t
```

Comment:
I think that using our coding style this still should be `Coq_config.t` , @Alizter ?

Reply:
Yes, but I think it would be too confusing to split this further into separate files. I'm fine with it like this.

Reply:
I think we should not split, but keep `Coq_config.t` (as is now on `main`) , and add a `Version` module so we have `Coq_config.Version.t` too, both on the same file. WDYT?

Reply:
Doesn't make a difference to me.